modules/react/button/lib/ToolbarDropdownButton.tsx

```diff
@@ -1,115 +1,107 @@
-} from '@workday/canvas-kit-react/common';
-import {ButtonColors} from './types';
-import {BaseButton} from './BaseButton';
+import {brand, system} from '@workday/canvas-tokens-web';
+import {calc, createStencil, handleCsProp, px2rem} from '@workday/canvas-kit-styling';
 import {chevronDownSmallIcon} from '@workday/canvas-system-icons-web';
+import {focusRing, createComponent} from '@workday/canvas-kit-react/common';
+import {systemIconStencil} from '@workday/canvas-kit-react/icon';
+
+import {BaseButton, buttonStencil} from './BaseButton';
 import {ToolbarIconButtonProps} from './ToolbarIconButton';
-import {brand} from '@workday/canvas-tokens-web';
 
 export interface ToolbarDropdownButtonProps
-  extends Omit<ToolbarIconButtonProps, 'toggled' | 'onToggleChange'>,
-    Themeable {}
+  extends Omit<ToolbarIconButtonProps, 'toggled' | 'onToggleChange'> {}
 
-const StyledToolbarDropdownButton = styled(BaseButton)<StyledType & ToolbarDropdownButtonProps>({
-  padding: space.zero,
-  minWidth: space.l,
-  width: 'auto',
-  height: space.l,
-  borderRadius: borderRadius.m,
-  gap: space.zero,
-  '& .wd-icon': {
-    display: 'inline-block',
-    width: 20,
-    height: 20,
-  },
-  '& .wdc-toolbar-dropdown-btn-arrow': {
-    margin: '0 2px 0 0',
-  },
-  '& .wdc-toolbar-dropdown-btn-custom-icon': {
-    marginLeft: `${space.xxxs}`,
-    marginRight: 0,
-    width: 18, // decrease the space between a custom icon and the chevron per design
+export const toolbarDropdownButtonStencil = createStencil({
+  extends: buttonStencil,
+  parts: {
+    chevron: 'toolbar-dropdown-btn-arrow',
+    customIcon: 'toolbar-dropdown-btn-custom-icon',
   },
-  '&:focus-visible, &.focus': {
-    ...focusRing({
-      width: 2,
-      separation: 0,
-      innerColor: 'transparent',
-      outerColor: brand.common.focusOutline,
-    }),
+  base: ({chevronPart, customIconPart}) => ({
+    padding: system.space.zero,
+    minWidth: system.space.x8,
+    gap: system.space.zero,
+    [buttonStencil.vars.borderRadius]: system.shape.x1,
+    [systemIconStencil.vars.color]: system.color.fg.muted.soft,
+
+    '&:focus-visible, &.focus': {
+      [buttonStencil.vars.background]: system.color.bg.transparent,
+      [systemIconStencil.vars.color]: system.color.fg.muted.soft,
+      ...focusRing({
+        width: 2,
+        separation: 0,
+        innerColor: system.color.border.transparent,
+        outerColor: brand.common.focusOutline,
+      }),
+    },
+
+    '&:hover, &.hover': {
+      [buttonStencil.vars.background]: system.color.bg.alt.default,
+      [systemIconStencil.vars.color]: system.color.fg.muted.stronger,
+    },
+
+    '&:active, &.active': {
+      [buttonStencil.vars.background]: system.color.bg.alt.stronger,
+      [systemIconStencil.vars.color]: system.color.fg.muted.stronger,
+    },
+
+    '&:disabled, &.disabled': {
+      [buttonStencil.vars.background]: system.color.bg.transparent,
+      [systemIconStencil.vars.color]: system.color.fg.disabled,
```

Comment:
@mannycarrera4 What do you think about changing this to `system.color.fg.disabled`? previous value is `soap600` and we don't have a token for this. I think it's reasonable to change to existing token instead.

Reply:
yep that works, I think we use that token in our Tabs component!